interp/constrextern.ml

```diff
@@ -486,7 +486,7 @@ and apply_notation_to_pattern ?loc gr ((subst,substlist),(no_implicit,nb_to_drop
   function
     | NotationRule (_,ntn as specific_ntn) ->
       begin
-        match availability_of_entry_coercion custom (fst ntn) with
+        match availability_of_entry_coercion custom InConstrEntrySomeLevel with
```

Comment:
@herbelin I'm not sure what to do here, this is the only place (with the one below) where the notation level is needed from a `NotationRule`.